jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/DelayedHandler.java

```diff
@@ -48,6 +75,21 @@ public DelayedHandler(Handler handler)
         super(handler);
     }
 
+    public long getMaxRetainedContentBytes()
+    {
+        return _maxRetainedContentBytes;
+    }
```

Comment:
Add getter/setter for `framingLength`.

Please also update the Jetty module and XML.